torch_xla/csrc/ops/device_data.cpp

```diff
@@ -0,0 +1,27 @@
+#include "ops/device_data.h"
+
+#include <sstream>
+
+#include "lowering_context.h"
+#include "ops/xla_ops.h"
+
+namespace torch_xla {
+namespace ir {
+namespace ops {
+
+DeviceData::DeviceData(std::shared_ptr<xla::ComputationClient::Data> data)
+    : Node(OpKind(XlaOps().device_data), {}), data_(std::move(data)) {}
```

Comment:
Couldn't we make operator kinds static fields of a class instead? `XlaOps().device_data` looks quite weird.

Reply:
Initialization order. Better not have static data whose construction requires calling other functions.
But that OpKind() caster need not to be there :)

Reply:
Ok, if we're not willing to bend this rule there isn't a better way.